runtime/modules/java/core/src/main/java/io/atlasmap/java/inspect/StringUtil.java

```diff
@@ -24,7 +24,7 @@ public static boolean isEmpty(String s) {
         return s == null || "".equals(s.trim());
```

Comment:
For me this is also a Java specific utilities and could be migrated into ClassHelper, WDYT @sunilmeddr ?

Reply:
All the string based utility methods are in the classStringUtil.java . This is one of the utility method.